python/integration-tests/function/main.tf

```diff
@@ -0,0 +1,22 @@
+resource "aws_lambda_layer_version" "opentelemetry_python_wrapper" {
+  layer_name          = var.sdk_layer_name
+  filename            = "../../src/build/layer.zip"
+  compatible_runtimes = ["python3.8"]
+  license_info        = "Apache-2.0"
+  source_code_hash    = filebase64sha256("../../src/build/layer.zip")
+}
+
+resource "aws_lambda_layer_version" "opentelemetry_collector" {
+  layer_name          = var.collector_layer_name
+  filename            = "../../../collector/build/collector-extension.zip"
+  compatible_runtimes = ["nodejs10.x", "nodejs12.x", "nodejs14.x"]
```

Comment:
that is ok since Lambda limits at most 5 compatible runtimes, but does not impact adding it to any other runtimes.